appveyor.yml

```diff
@@ -13,6 +14,9 @@ cache:
 
 install:
   - SET PATH=C:\%ruby_version%\bin;%PATH%
+  - gem uninstall bundler -a -x -I
+  - gem update --system 2.7.8
```

Comment:
@iggant thanks a lot! btw can we push the version to 2.7.9, which is known to be the last rubygems version before 3.0. We have to keep using that until we drop support for Ruby < 2.3.

But still bundler version needs to be < 2, as long as we support Rails < 5.

Reply:
done